elastic/beats
# [Heartbeat] Update gval to latest version to improve Heartbeat JSON parsing

## What does this PR do?

Updates `gval` module to the latest version.

## Why is it important?

Updates `gval` to latest version with the hope that parsing will be improved for users.

## Checklist



~~- [ ] My code follows the style guidelines of this project~~
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~

## Author's Checklist

- [ ] Make sure JSON functions are evaluated properly

## How to test this PR locally

You can create a monitor like:

```yaml
- type: http
  id: sendgrid-smtp-status
  name: Check Sendgrid SMTP
  fields:
    component: "sendgrid-smtp-status"
  schedule: '@every 10s'
  response.include_body: always
  tags: ["sendgrid-smtp"]
  urls:
    - https://status.sendgrid.com/api/v2/components.json
  check.request.method: GET
  check.response:
    status: [200]
    json:
      - description: Check Sendgrid SMTP status
        expression: '$.components[?(@.name == "SMTP" && @.status == "operational")] != []'
```

It should have a status of `up`

## Use cases

Ensure we have the latest version of the JSON parser we use.

## Screenshots



## Logs
